### PR TITLE
Add ability to publish JAR SDKs

### PR DESCRIPTION
**Pull request by vkryachko**

### CODE REVIEW

File: buildSrc/src/main/java/com/google/firebase/gradle/plugins/Tasks.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.gradle.plugins
+
+const val JAVADOC_TASK_NAME = "firebaseJavadoc"
```
Comment (rlazo) [inline buildSrc/src/main/java/com/google/firebase/gradle/plugins/Tasks.kt:17]: where is this used?

Comment (vkryachko) [reply buildSrc/src/main/java/com/google/firebase/gradle/plugins/Tasks.kt:17]: Looks like it's only needed in the release repo, will move it there.

**Pull request merged**